src/index.js

```diff
@@ -115,6 +115,7 @@ module.exports = {
   VoiceRegion: require('./structures/VoiceRegion'),
   VoiceState: require('./structures/VoiceState'),
   Webhook: require('./structures/Webhook'),
+  Widget: require('./structures/Widget'),
```

Comment:
WidgetMember should also be exported:
```suggestion
  Widget: require('./structures/Widget'),
  WidgetMember: require('./structures/WidgetMember'),
```